tests/Makefile

```diff
@@ -21,6 +21,7 @@ ALL_TEST_DIRS += vale
 ALL_TEST_DIRS += hacl
 ALL_TEST_DIRS += simple_hello
 ALL_TEST_DIRS += dune_hello
+ALL_TEST_DIRS += validation-time
```

Comment:
The new single test here is not related to time at all? I think you can just drop `Text.LexemeFL.fst` in `tests/micro-benchmarks` and not add any new Makefiles.

Reply:
validation-time is the name of the phase right? Nothing to do with time. I didn't put it in micro-benchmarks as it is a correctness test. For Unix I added a run-time test directory that I'm working on right now.

Reply:
`micro-benchmarks` is a bad name really.. we use that directory for a whole bunch of unit tests. I'll rename it soon.